homeassistant/components/netatmo/__init__.py

```diff
@@ -62,6 +64,8 @@ async def async_setup(hass: HomeAssistant, config: dict):
     hass.data[DOMAIN][DATA_DEVICE_IDS] = {}
     hass.data[DOMAIN][DATA_SCHEDULES] = {}
     hass.data[DOMAIN][DATA_HOMES] = {}
+    hass.data[DOMAIN][DATA_EVENTS] = {}
```

Comment:
Why is it that we don't scope this to the config entry ID ?

Reply:
So far there is only once Netatmo instance supported. Was easy and not criticized yet. But I can work on that.

Reply:
Let's keep that for a new PR 👍